dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/bundle/DefaultTrackerBundleService.java

```diff
@@ -442,90 +439,41 @@ private TrackerTypeReport handleRelationships( Session session, TrackerBundle bu
     // -----------------------------------------------------------------------------------
 
     private void handleTrackedEntityAttributeValues( Session session, TrackerPreheat preheat,
-        List<Attribute> attributes, TrackedEntityInstance trackedEntityInstance )
+        List<Attribute> payloadAttributes, TrackedEntityInstance trackedEntityInstance )
     {
-        List<TrackedEntityAttributeValue> attributeValues = new ArrayList<>();
-        List<String> attributeValuesForDeletion = new ArrayList<>();
-
-        List<String> assignedFileResources = new ArrayList<>();
-        List<String> unassignedFileResources = new ArrayList<>();
-
-        Map<String, TrackedEntityAttributeValue> attributeValueMap = trackedEntityInstance
+        Map<String, TrackedEntityAttributeValue> attributeValueDBMap = trackedEntityInstance
             .getTrackedEntityAttributeValues()
             .stream()
             .collect( Collectors.toMap( teav -> teav.getAttribute().getUid(),
                 trackedEntityAttributeValue -> trackedEntityAttributeValue ) );
```

Comment:
minor: you can use `Function:identity()` instead of `trackedEntityAttributeValue -> trackedEntityAttributeValue`